docs/USAGE-GUIDE.md

```diff
@@ -568,6 +568,9 @@ The following chart maps the security policy version to the signature scheme sup
 |   "20190802"   |      X       |     X    |      X        |    X     |
 |   "20200207"   |      X       |     X    |      X        |    X     |
 
+Note that currently the default_tls13 security policy supports SHA-1 legacy algorithms in Certificate Verify messages,
+but it will not support SHA-1 Legacy algorithms in certificate signatures.
```

Comment:
You might want to make this clearer: default_tls13 supports SHA-1 in CertificateVerify **when using TLS1.2**. Never in TLS1.3.